### PR TITLE
Provide precompiled bundle

### PR DESCRIPTION
**Pull request by Nachtalb**

### CODE REVIEW

File: ftw/solr/upgrades/20191211175444_provide_precompiled_bundle/registry.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<registry>
+    <records prefix="plone.bundles/ftw-solr-resources"
```
Comment (jone) [inline ftw/solr/upgrades/20191211175444_provide_precompiled_bundle/registry.xml]: Please also remove the individual resources
```xml
    <records prefix="plone.resources/ftwsolr_liveserch" interface="Products.CMFPlone.interfaces.IResourceRegistry" remove="True" />
    <records prefix="plone.resources/ftwsolr_facets" interface="Products.CMFPlone.interfaces.IResourceRegistry" />
    <records prefix="plone.resources/ftwsolr_jquery_history" interface="Products.CMFPlone.interfaces.IResourceRegistry" />
    <records prefix="plone.resources/ftwsolr_jquery_legazy_highlight" interface="Products.CMFPlone.interfaces.IResourceRegistry" />
    <records prefix="plone.resources/ftwsolr_search" interface="Products.CMFPlone.interfaces.IResourceRegistry" />

```

File: yarn-error.log [hunk 2]
```diff
@@ -0,0 +1,346 @@
+Arguments: 
+  /usr/local/Cellar/node/13.2.0/bin/node /usr/local/bin/yarn build
```
Comment (jone) [inline yarn-error.log]: Please add `yarn-error.log` to the git ignrores.

File: ftw/solr/upgrades/20191211175444_provide_precompiled_bundle/registry.xml (hunk 1 above)
Comment (Nachtalb) [reply ftw/solr/upgrades/20191211175444_provide_precompiled_bundle/registry.xml]: Done

File: yarn-error.log (hunk 2 above)
Comment (Nachtalb) [reply yarn-error.log]: Done

**Comment by Nachtalb** [comment]
Rebased onto latest master.

**Pull request merged**